lib/smart_proxy_reports/spooled_http_client.rb

```diff
@@ -91,8 +96,8 @@ def wakeup
       @worker.wakeup if @worker
     end
```

Comment:
Note for myself: ensure this is still HTTP persistent connection after this refactoring.